reana_server/rest/workflows.py

```diff
@@ -226,19 +234,24 @@ def get_workflows(user, **kwargs):  # noqa
                 "message": "Something went wrong."
               }
     """
+    import pdb; pdb.set_trace()
     try:
         type_ = request.args.get("type", "batch")
         search = request.args.get("search")
         sort = request.args.get("sort", "desc")
         status = request.args.getlist("status")
         verbose = json.loads(request.args.get("verbose", "false").lower())
+        include_progress = request.args.get("include_progress", "")
+        include_workspace_size = request.args.get("include_workspace_size", "")
```

Comment:
Instead of manually unwrapping the value here, maybe you can use the marshmallow schema in line 58? So validation is easier.